docs/source/developer-guidelines/contributing.md

```diff
@@ -0,0 +1,60 @@
+---
+myst:
+  html_meta:
+    "description": "A guide on how to contribute to the Plone's community"
+    "property=og:description": "A guide on how to contribute to the Plone's community"
+    "property=og:title": "Guidelines"
+    "keywords": "Volto, Plone, frontend, React, guidelines"
+---
+
+# Guidelines for Contributing
+
+You probably came here by clicking one of the ‘guidelines for contributing’ links on
+GitHub.
+You probably have an issue to report or you want to create a pull request.
+
+Thanks a lot! Let’s bring you up to speed with the minimum you need to know to start
+contributing.
+
+## Create an issue
+
+If you know the issue is for Volto, you can add an issue in [Volto issue tracker](https://github.com/plone/volto/issues).
+When in doubt, create one in the [CMFPlone issue tracker](https://github.com/plone/Products.CMFPlone/issues "CMFPlone issue tracker").
+
+Please specify a few things:
+
+- What steps reproduce the problem?
+- What do you expect when you do that?
+- What happens instead?
+- Which Plone version are you using?
+
+If it is a visual issue, can you add a screenshot?
+
+## Create a pull request
+
+Legally, you can NOT contribute code unless you have signed the contributor agreement.
+This means that we can NOT accept pull requests from you unless this is done, so please
+don’t put the code reviewers at risk and do it anyways.
+
+For new features, an addition to documentation is probably needed.
+
+If the feature includes a breaking change, you should also add the breaking and how to
+upgrade in the [upgrade guide](../upgrade-guide/index.md).
+
+All text that can be shown in a browser must be translatable. Please mark all such
+strings as translatable as defined in the [i18n guide](../recipes/i18n.md)
+
```

Comment:
Replace everything below this comment with the following.

```
## Code Quality

All pull requests must pass tests, documentation builds, and other code quality checks.
Contributors are strongly encouraged to run these checks locally before creating a pull request.
These checks are enforced automatically on every pull request, so you might as well save time and frustration by doing these checks locally first.

Specifically:

-   {doc}`./linting`
-   {doc}`./testing`
-   {doc}`./acceptance-tests`
```

If after reading this you become hesitant, don't worry.
You can always create a pull request, mark it as "Draft", and improve the above points later, requesting help from the community.

Welcome to the Plone community, and thank you for contributing!